Lesson3/Lesson3/Program.cs

```diff
@@ -250,7 +253,7 @@
             break;
             
         }
-        //result += number1 + number2;
+        
         int otvet = nmbr1 + result + number2;
```

Comment:
This logic should not be included into cycle. If the variable "otvet" is declared inside the cycle, then you cannot use it outside the cycle (when the cycle is finished)